gcc/rust/lex/rust-lex.h

```diff
@@ -217,9 +219,9 @@ class Lexer
 
     Codepoint next_codepoint ()
     {
-      uint8_t input = next_byte ();
+      uint32_t input = next_byte ();
 
-      if ((int8_t) input == EOF)
+      if ((int32_t) input == EOF)
 	return Codepoint::eof ();
```

Comment:
Changed `input` from `uint8_t` to `uint32_t` so as to differentiate `0xff` and EOF.

Reply:
just a bugfix